pyface/ui/qt4/util/gui_test_assistant.py

```diff
@@ -12,8 +12,8 @@
 
 import mock
 
+from pyface.gui import GUI
```

Comment:
Please revert this import to the original importing from `qt4`. Importing from `pyface.gui` from inside the `ui.qt4` submodule is not right (and probably not totally safe)